zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/errorhandling/Error.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.errorhandling;
+
+public record Error(String message, String type, int status) {}
```

Comment:
🔧 I think here some more java doc would be beneficial as well, why we have it where is it used or so